.gitignore

```diff
@@ -244,3 +244,6 @@ $RECYCLE.BIN/
 
 # Windows shortcuts
 *.lnk
+
+# Generated JSDOCs
```

Comment:
We are pushing the docs in the master branch, until we solve that, let's not add this to the ignore.